frontend/src/app/notification/page.tsx

```diff
@@ -0,0 +1,33 @@
+import { Anchor, Center,Group,Stack,Text, Title } from "@mantine/core"
+import { FC } from "react"
+
+const WordsOfCaution: FC = () => {
+    return ( 
+        <Group justify="center">
+        <Stack align="center" w="60%">
+            <Title mt="lg">🚨Words of caution to our early user adopters🚨</Title>
+            <Text>
+                Bug Buster is in alpha stage and this means that its development is heavily
+                active thus changes (new features, improvements and fixes) can come anytime.
+            </Text>
+            <Text>
+                In regards to security, althought Bug Buster is built on top of Cartesi Rollups SDK
+                and its development has been done with care, we cannot guarantee that the platform 
+                is free of bugs or vulnerabilities at this stage.
+            </Text>
+            <Text>
+                To avoid any potential loss of funds, we recommend that you use the platform with scaution, and avoid using it with large amounts of funds.
```

Comment:
```suggestion
                To avoid any potential loss of funds, we recommend that you use the platform with caution, and avoid depositing large amounts of funds.
```